styles/git-blame.less

```diff
@@ -7,7 +7,6 @@
 
 @git-blame-bg-color: @base-background-color;
 
-atom-text-editor::shadow .git-blame-mount,
```

Comment:
The original issue shows the automatic translation that atom will do. It's a little hard to see in the way it was reported, but they have the original shadow form of the selector followed by `=>` and then the automatically transpiled version. Per the docs you are correct, though they do reference it down here: http://flight-manual.atom.io/shadow-dom/sections/removing-shadow-dom-styles/#i-followed-the-guide-but-now-my-styling-is-broken

It is a bit unclear which is preferred, but since it was automatically adding the `.editor` before it is fully deprecated, I think it would be good to keep that as an option for ensuring specificity is high (though I doubt that there would be a conflict with `.git-blame-mount`)